app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def update
+    user = User.find_by(id: params[:id])
+    return render json: { errors: ['User not found'] }, status: :not_found unless user
+
+    if user.update(users_params)
```

Comment:
Given that we've authenticated the request and established current_user, would it be appropriate to use current_user.update? Please let me know if I'm overlooking any considerations.